stable/kuberhealthy/templates/deployment.yaml

```diff
@@ -20,6 +20,12 @@ spec:
     type: RollingUpdate
   template:
     metadata:
+      {{- if .Values.deployment.podAnnotations }}
+      annotations:
+      {{- range $key, $value := .Values.deployment.podAnnotations }}
```

Comment:
You should just use the toYaml function in helm

Reply:
iterating allows to add the quote function - so I think it is OK